src/powfacpy/case_studies.py

```diff
@@ -281,7 +281,7 @@ def get_study_cases(self, conditions, return_case_numbers=False):
 
   def get_study_cases_from_string(self, conditions: str, return_case_numbers=False):
     """This method is another convenient way to get study cases according to conditions.
-    The condistions are a simple lambda function argument string (see example below).
+    The conditions are a simple lambda function argument string (see example below).
     This method is more convenient but less save than get_study_cases because the
     conditions string is evaluated and a lambda function is created from it. Using
```

Comment:
i would add something like 'although used here, ...' 
because it sounds like you dont use eval() and also dont recommend it